README.md

```diff
@@ -16,9 +16,23 @@ The DataView Demo application includes an example of using this helper:
 
 https://github.com/trevordevore/dataview_demo
 
+## Including File Browser in your project
+To include the File Browser Dataview control in your project (using git):
+1. Submodule the File Browser Dataview control https://github.com/trevordevore/levurehelper-file_browser to app/helpers/levurehelper-file_browser
+2. Submodule the DataView helper https://github.com/trevordevore/levurehelper-dataview to app/helpers/levurehelper-dataview
+3. Submodule the DataView Tree helper https://github.com/trevordevore/levurehelper-dataview_tree to app/helpers/levurehelper-dataview-tree
+4. Add the following keys to your app.yml file
+```
+helpers:
+  - folder: ./helpers
+  - filename: "[[FRAMEWORK]]/helpers/levurehelper-file_browser"
+  - filename: "[[FRAMEWORK]]/helpers/levurehelper-dataview"
+  - filename: "[[FRAMEWORK]]/helpers/levurehelper-dataview_tree"
```

Comment:
@macMikey These helper paths are incorrect as the helpers are not distributed with the Levure framework. You don't need to modify the app.yml file at all since the helpers are added to the ./app/helpers folder. The default app.yml file includes all helpers in the ./app/helpers folder.